SS_versioninfo_330safe.tpl

```diff
@@ -5,9 +5,13 @@ DATA_SECTION
 !!//  SS_Label_Section_1.0 #DATA_SECTION
 
 !!//  SS_Label_Info_1.1.1  #Create string with version info
-!!version_info+="#V3.30.17.01Beta;_2021_08_24;_safe;_Stock_Synthesis_by_Richard_Methot_(NOAA)_using_ADMB_12.3";
-!!version_info+="#Stock Synthesis (SS) is a work of the U.S. Government and is not subject to copyright protection in the United States.";
-!!version_info+="#Foreign copyrights may apply. See copyright.txt for more information.";
-!!version_info2+="#_user_support_available_at:NMFS.Stock.Synthesis@noaa.gov";
-!!version_info3+="#_user_info_available_at:https://vlab.noaa.gov/group/stock-synthesis";
+!!version_info+="#V3.30.18.Beta;_not_opt;_compile_date:_";
```

Comment:
I think "not_opt" could be confusing for some users; maybe stick with "safe" or something like "standard"?